main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/ministryofjustice/analytical-platform-auth0-user-exports/auth"
+	"log"
+	"os"
+	"time"
+
+	session2 "github.com/aws/aws-sdk-go/aws/session"
+	env "github.com/ministryofjustice/analytical-platform-auth0-user-exports/default_env_value"
+	connection "github.com/ministryofjustice/analytical-platform-auth0-user-exports/get_connection"
+	"github.com/ministryofjustice/analytical-platform-auth0-user-exports/job"
+)
```

Comment:
`import`s are a bit all over the place ☺️ 

from https://github.com/golang/go/wiki/CodeReviewComments#imports

> Avoid renaming imports except to avoid a name collision; good package names should not require renaming. In the event of collision, prefer to rename the most local or project-specific import.
>
> Imports are organized in groups, with blank lines between them. The standard library packages are always in the first group.